llvm/test/CodeGen/LoongArch/lsx/vec-sext.ll

```diff
@@ -0,0 +1,322 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc --mtriple=loongarch64 --mattr=+lsx %s -o - | FileCheck %s
+
+
+define void @load_sext_2i8_to_2i64(ptr%ptr, ptr%dst) {
```

Comment:
```suggestion
define void @load_sext_2i8_to_2i64(ptr %ptr, ptr %dst) {
```

Reply:
It looks like this change was unintentionally lost during a rebase. Could you check?